cvat/apps/engine/views.py

```diff
@@ -242,24 +258,37 @@ def annotations(self, request, pk):
                 except (AttributeError, IntegrityError) as e:
                     return Response(data=str(e), status=status.HTTP_400_BAD_REQUEST)
                 return Response(data)
+        elif request.method == 'POST':
+            return upload_anno_proxy(
+                request=request,
+                rq_id="{}@/api/v1/tasks/{}/annotations/upload".format(request.user, pk),
+                rq_func=annotation.upload_task_anno,
+                pk=pk,
+            )
 
     @action(detail=True, methods=['GET'], serializer_class=None,
         url_path='annotations/(?P<filename>[^/]+)')
     def dump(self, request, pk, filename):
         filename = re.sub(r'[\\/*?:"<>|]', '_', filename)
-        queue = django_rq.get_queue("default")
         username = request.user.username
         db_task = self.get_object()
         timestamp = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
-        file_ext = request.query_params.get("format", "xml")
         action = request.query_params.get("action")
         if action not in [None, "download"]:
             raise serializers.ValidationError(
                 "Please specify a correct 'action' for the request")
 
+        dump_format = request.query_params.get("dump_format", "")
```

Comment:
Why not `format`? Why `dump?dump_format=...`?

Reply:
`format` word is used by rest framework by defaults https://www.django-rest-framework.org/api-guide/format-suffixes/#query-parameter-formats. Should we modify name or disable it?